repos/fdbt-site/src/pages/returnService.tsx

```diff
@@ -0,0 +1,146 @@
+                            className="govuk-select"
+                            id="service"
+                            name="serviceId"
+                            defaultValue={error.length > 0 && error[0].userInput ? error[0].userInput : undefined}
+                        >
+                            <option value="" disabled>
+                                Select One
+                            </option>
+                            {services
+                                .filter((service) => {
+                                    return service.id != selectedServiceId;
+                                })
+                                .map((service) => (
+                                    <option key={`${service.id}`} value={`${service.id}`} className="service-option">
+                                        {service.lineName} {service.origin || 'N/A'} - {service.destination || 'N/A'}{' '}
+                                        (Start date {service.startDate})
+                                    </option>
+                                ))}
+                        </select>
+                    </FormElementWrapper>
+                    <span className="govuk-hint hint-text" id="traveline-hint">
+                        This data is taken from the <b>Bus Open Data Service (BODS)</b>. If the service you are looking
+                        for is not listed, contact the BODS help desk for advice <a href="/contact">here</a>
+                    </span>
+                </div>
+                <input type="hidden" name="selectedServiceId" value={selectedServiceId} />
+                <input type="submit" value="Continue" id="continue-button" className="govuk-button" />
+            </>
+        </CsrfForm>
+    </TwoThirdsLayout>
+);
+
+export const getServerSideProps = async (ctx: NextPageContextWithSession): Promise<{ props: ReturnServiceProps }> => {
+    const csrfToken = getCsrfToken(ctx);
+
+    const ticket = getSessionAttribute(ctx.req, MATCHING_JSON_ATTRIBUTE);
+    const matchingJsonMetaData = getSessionAttribute(ctx.req, MATCHING_JSON_META_DATA_ATTRIBUTE);
+    const nocCode = getAndValidateNoc(ctx);
+    const selectedServiceId = Number(ctx.query.selectedServiceId);
+
+    const operatorAttribute = getSessionAttribute(ctx.req, OPERATOR_ATTRIBUTE);
+
+    if (!ticket || !matchingJsonMetaData) {
+        throw new Error('Ticket details not found');
+    }
+    if (!selectedServiceId) {
+        throw new Error('Added service is missing');
+    }
+
+    if (!isReturnTicket(ticket)) {
+        throw new Error('Ticket should be return type');
+    }
+
+    const services = await getServicesByNocCodeAndDataSource(nocCode, 'bods');
+
+    if (services.length === 0) {
+        if (ctx.res) {
+            redirectTo(ctx.res, '/noServices');
+        } else {
+            throw new Error(`No services found in BODS for NOC Code: ${nocCode}`);
+        }
+    }
+
+    const error: ErrorInfo[] = [];
```

Comment:
We can remove this and just change the below to `errors: []`

Reply:
removed this line